CHANGES.rst

```diff
@@ -26,6 +26,8 @@ Improvements
 - Allow filtering by location in room booking (:issue:`4291`, :pr:`5622`,
   thanks :user:`mindouro`)
 - Add button to adapt column widths in paper & contribution lists (:pr:`5642`)
+- Add a :data:`locked_fields` setting to ``indico.conf`` to prevent non-admin
```

Comment:
I don't think the `:data:` will help here since it's not a top-level key in the rst file. But maybe mention "to identity provider settings" there